wwwroot/images/icons/compare.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="14px" viewBox="0 0 18 14" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 63.1 (92452) - https://sketch.com -->
+    <title>compare</title>
+    <desc>Created with Sketch.</desc>
+    <g id="Symbols" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Assets/UI/Compare/Hover/Open" transform="translate(-10.000000, -9.000000)" fill="#4D5766">
```

Comment:
Hardcoded fill, see `user.svg` for an example of the output from a `svgo`ified svg, I use this once I've cleared out the fill colour: https://jakearchibald.github.io/svgomg/